### PR TITLE
Sometimes in priceapi tokens get lost when deploying - fix that

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: lib/redis/semaphore.rb [hunk 1]
```diff
@@ -143,6 +143,11 @@ def release_stale_locks!
             signal(token)
           end
         end
+        if (tokens = all_tokens.map(&:to_i)).count < @resource_count
```
Comment (jannishuebl) [inline lib/redis/semaphore.rb:146]: saved one iteration :D

**Comment by jannishuebl** [comment]
:+1:

**Review by jan — COMMENTED** [review]
Can you provide a spec for the issue that you are fixing?

**Comment by ghost** [comment]
@jan I did now

**Pull request merged**